backend/core/src/shared/email/email.service.spec.ts

```diff
@@ -52,7 +52,8 @@ describe("EmailService", () => {
           provide: REQUEST,
           useValue: {
             get: () => {
-              return { "county-code": CountyCode.alameda }
+              // This header must be the string value of the CountyCode key.
+              return "alameda"
```

Comment:
Should this be Detroit? Also why did we delete "country-code" and no longer use the CountryCode object to fetch the string?

Reply:
This had to be alameda because the Archer listing referred to in this test has a CountyCode of Alameda. The object is not used because of the mock - the request.get is used as the key for the CountyCode and should actually be the key instead of an object that contains the key.

Reply:
Got it thanks!